tokio-util/src/io/copy_to_bytes.rs

```diff
@@ -0,0 +1,68 @@
+use bytes::Bytes;
+use futures_sink::Sink;
+use pin_project_lite::pin_project;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+pin_project! {
+    /// A helper structure which wraps a `Sink<&'a [u8]>` and converts it into
```

Comment:
Really minor nit - I'd remove `structure` here and reflow. It's obvious that it's a structure from the type definition (which is visible in `rustdoc` output).

Reply:
Minor nitpicks are just as important